src/test/java/uk/gov/justice/digital/delius/service/AttendanceServiceTest.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.justice.digital.delius.service;
+
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+import static uk.gov.justice.digital.delius.service.AttendanceService.forEntityBoolean;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.justice.digital.delius.data.api.Attendance;
+import uk.gov.justice.digital.delius.jpa.standard.entity.Contact;
+import uk.gov.justice.digital.delius.jpa.standard.entity.ContactOutcomeType;
+import uk.gov.justice.digital.delius.jpa.standard.entity.ContactType;
+import uk.gov.justice.digital.delius.jpa.standard.repository.ContactRepository;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AttendanceServiceTest {
+
+    public static final Long SOME_CONTACT_ID = 1234L;
+    public static final Long SOME_EVENT_ID = 5435L;
+    public static final String CONTACT_TYPE_CODE = "CT-CODE";
+    public static final String CONTACT_TYPE_DESC = "CT Description";
+    private static final String OUTCOME = "COT Description";
+
+    @Mock
+    private ContactRepository contactRepository;
+
+    @InjectMocks
+    private AttendanceService attendanceService;
+
+    @Test
+    public void getAttendancesForEventNoneAvailable() {
+
+        when(contactRepository.findByEventIdEnforcement(eq(SOME_EVENT_ID), any(LocalDate.class))).thenReturn(Collections.emptyList());
+
+        assertTrue(attendanceService.getContactsForEvent(SOME_EVENT_ID, LocalDate.now()).isEmpty());
+
+        verify(contactRepository).findByEventIdEnforcement(eq(SOME_EVENT_ID), any(LocalDate.class));
+        verifyNoMoreInteractions(contactRepository);
+    }
+
+    @SuppressWarnings("OptionalGetWithoutIsPresent")
```

Comment:
see previous comment about `orElseThrow()`

Reply:
It's test so won't happen unless test is failing in which case the test will fail.